pkg/filterlist/tagfilterlist.go

```diff
@@ -0,0 +1,32 @@
+package filterlist
```

Comment:
Do we plan to share this code with other places? If not, let's place it under `aggregator` (or `aggregator/internal/filterlist`).